static/app/components/forms/controls/rangeSlider/index.tsx

```diff
@@ -1,4 +1,4 @@
-import {forwardRef as reactFowardRef, useEffect, useState} from 'react';
+import {forwardRef as reactForwardRef, useEffect, useState} from 'react';
```

Comment:
in this case, we will have to update this line too 
https://github.com/getsentry/sentry/blob/9a6320baa48450d9e7fec4a530fe9cfe3110a43e/static/app/components/forms/controls/rangeSlider/index.tsx#L209

Reply:
Oh. This project seems to have a whitespace rule which is forcing VSCode to rewrite the entire file when I choose `save`. So I tried "manually" fixing the thing up, which is how I (I'm human, this is the proof!) messed up.